projects/prometheus/build.sh

```diff
@@ -15,6 +15,9 @@
 #
 ################################################################################
 
+# Required to avoid 'working directory is not part of a module' error.
+cd $GOPATH/src/github.com/prometheus/prometheus
```

Comment:
I think you need to move this to
WORKDIR $GOPATH/src/github.com/prometheus/prometheus in Dockerfile
that way local reproduction using infra/helper.py works (depends on WORKDIR)

Reply:
Thanks, I have updated the PR.